### DISCUSSION TITLE
Is it possible to use Microsoft.EntityFrameworkCore.Sqlite in uno wasm?

### DISCUSSION THREAD

**Discussion by JoacimWall** [unoplatform/uno#14560] [Q&A]
Hi 
Working on a project that are going to target mobile iOS and Droid and Wasm. I can get the Microsoft.EntityFrameworkCore.Sqlite to work on mobile but not  the Wasm project. 
If I do wasm with Blazor it works.  I use the SqliteWasmHelper nuget in the Blazor project.

//Joacim

**Comment by jeromelaban** [comment]
There's a sample here: https://github.com/unoplatform/Uno.SQLitePCLRaw.Wasm
